web/src/settings_components.ts

```diff
@@ -910,3 +912,23 @@ function enable_or_disable_save_button($subsection_elem: JQuery): void {
 
     $subsection_elem.find(".subsection-changes-save button").prop("disabled", disable_save_btn);
 }
+
+export function initialize_disable_btn_hint_popover(
+    $btn_wrapper: JQuery,
+    hint_text: string | undefined,
+    opts: Partial<Props>,
```

Comment:
This is probably better spelled `tippy.Props`, since `Props` is not meaningful globally in our codebase.